### PR TITLE
Add description about reproducible build

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

Added a description of how to create a reproducible build according to the requirement of OpenSSF Best Practicies

## Type of change

- [x] Documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -7,7 +7,8 @@
     2.2 [Vulnerability Report](#22-vulnerability-report)  
     2.3 [Security Disclosure](#23-security-disclosure)  
 3. [Security requrements](#3-security_requirements)
-4. [Security Software life cycle processes](#3-security-software-life-cycle-processes)
+4. [Security Software life cycle processes](#4-security-software-life-cycle-processes)
+5. []()
```
Comment (t25kim) [inline .github/SECURITY.md:11]: Is it correct?

Comment (tdrozdovsky) [reply .github/SECURITY.md]: Fixed! Done!

**Review by t25kim — APPROVED** [review]
LGTM Thank you very much!

**Review by MoonkiHong — APPROVED** [review]
Looks good to me.

**Review by suresh-lc — APPROVED** [review]
LGTM

**Pull request merged**